lib/beaker-hostgenerator/data/vmpooler.rb

```diff
@@ -35,6 +35,10 @@ module Vmpooler
           'platform' => 'el-7-x86_64',
           'template' => 'centos-7-x86_64'
         },
+        'cumulus2-64' => {
```

Comment:
should this be `cumulus25-64` in case we need to support cumulus 2.6.x in the future?